distributed/cli/tests/test_dask_submit.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import print_function, division, absolute_import
+from distributed.utils_test import valid_python_script, popen, invalid_python_script
+
+def test_dask_submit_cli(valid_python_script, invalid_python_script):
+    with popen(['dask-scheduler', '--port', '8989']) as sched:
+        with popen(['dask-submit', '127.0.0.1:8788', str(valid_python_script)]) as submit:
+            for x in range(5):
+                line = submit.stdout.readline()
+                if b'hello world!\\n' in line:
+                    break
+
+        with popen(['dask-submit', '127.0.0.1:8788', str(invalid_python_script)]) as submit:
+            for x in range(5):
+                line = submit.stdout.readline()
+                if b'Traceback' in line:
+                    break
+
+
+def test_submit_a_distributed_script(executor_contract_script):
```

Comment:
Submitting a script on local machine without RemoteClient works. I am beginning to wonder if `subprocess.Popen` inside a coroutine is doing something interesting. The scheduler logs start showing stuff like:

```
bokeh.command.subcommands.serve - INFO - Starting Bokeh server on port 8787 with applications at paths ['/status', '/tasks']
distributed.core - INFO - Connection from 10.34.13.61:50499 to Scheduler
distributed.core - INFO - Connection from 10.34.13.61:50500 to Scheduler
distributed.core - INFO - Connection from 10.34.13.61:51052 to RemoteClient
bk_script_5585e068b2024763b0e64497d612d069 - WARNING - workers http route failed
bk_script_5585e068b2024763b0e64497d612d069 - WARNING - http route tasks failed

```